### PR TITLE
Add README.md for the IPA-Tuura user federation

### LINKED ISSUE #35563: Add a README for the IPA-Tuura federation
### Description

We need to include a README.md for the IPA-Tuura user federation with instructions for devs to run and configure the bridge service so the federation can be used/tested.

### PR DESCRIPTION
**Pull request by sguilhen**
Closes #35563

### CODE REVIEW

File: federation/ipatuura/README.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+3- the `"ldap_tls_cert"` is not actually used in this setup, just set any value
+
+4- `"keycloak_hostname"` must resolve to the host running keycloak (see step above to ensure it was added to `/etc/hosts`)
+
+## Setup the IPA-Tuura user federation in keycloak
+
+First start keycloak with the `ipa-tuura-federation` feature enabled:
+
+```
+$ ./kc.sh start-dev --debug --db=postgres --db-username=<youruser> --db-password=<yourpass> --features=ipa-tuura-federation
```
Comment (pedroigor) [inline federation/ipatuura/README.md:195]:
```suggestion
$ ./kc.sh start-dev --features=ipa-tuura-federation
```

Comment (pedroigor) [reply federation/ipatuura/README.md:195]: The suggestion above is to make it more clear that the only thing you need is the feature enabled.

**Review by pedroigor — COMMENTED** [review]
Nice! Only a single suggestion.

Looking forward to see more about IPA Tuura integration!

File: federation/ipatuura/README.md (hunk 1 above)
Comment (sguilhen) [reply federation/ipatuura/README.md:195]: Agree, will send update!

Comment (sguilhen) [reply federation/ipatuura/README.md]: Done!

**Review by pedroigor — APPROVED** [review]

**Pull request merged**